bids/analysis/analysis.py

```diff
@@ -12,15 +12,18 @@
 
 
 class Analysis(object):
-    ''' Represents an entire BIDS-Model analysis.
-
-    Args:
-        layout (BIDSLayout, str): A BIDSLayout instance or path to pass on
-            to the BIDSLayout initializer.
-        model (str or dict): a BIDS model specification. Can either be a
-            string giving the path of the JSON model spec, or an already-loaded
-            dict containing the model info.
-    '''
+    """Represents an entire BIDS-Model analysis.
+
+    Parameters
+    ----------
+    layout : :obj:`bids.layout.BIDSLayout` or str
```

Comment:
Nice. I guess we don't have a test that renders the docs for inspection?

Reply:
No, I don't believe there are any tests for the docs. It's possible to build the docs in CI and then push them to RTD, which would be nice, but I think that would be a separate thing. I also have no clue how to do it.

Reply:
Okay. I'll build it locally to look. I'll open an issue to build the docs in CI so PRs like this can be more easily inspected (and we can get warnings when we break the build).